go.mod

```diff
@@ -35,7 +36,7 @@ replace (
 	github.com/containernetworking/cni => github.com/containernetworking/cni v0.8.1
 	github.com/containers/buildah => github.com/containers/buildah v1.23.1
 	github.com/opencontainers/image-spec => github.com/opencontainers/image-spec v1.0.2-0.20210819154149-5ad6f50d6283
-	github.com/project-flotta/flotta-operator v0.0.2-0.20220314112640-9186994c5b6e => github.com/gciavarrini/flotta-operator v0.0.0-20220317111214-7a6ac749cf71
+	github.com/project-flotta/flotta-operator => github.com/machacekondra/k4e-operator v0.0.2-0.20220315101727-9b44be993612
```

Comment:
Is there anything missing on `github.com/project-flotta/flotta-operator` ?

Reply:
This one: https://github.com/project-flotta/flotta-operator/pull/137/files